protocol/git.md

```diff
@@ -0,0 +1,85 @@
+Git Protocol
+============
+
+A guide for programming within version control.
+
+Maintain a Repo
+---------------
+
+* Avoid including files in source control that are specific to your
+  development machine or process by adding their formats to the .gitignore file.
+* Delete local and remote feature branches after merging.
+* Perform all new work in a feature branch.
+* Rebase frequently to incorporate upstream changes.
+* Use a [pull request] for code reviews.
+
+[pull request]: https://help.github.com/articles/using-pull-requests/
+
+Write a Feature
+---------------
+
+For all new development work, create a local feature branch based off develop.
+Be sure to include the JIRA ticket in your feature name (example: DHH-123).
```

Comment:
That's very true. I just looked at the documentation again, and it turns out that JIRA should pick up on either a commit or a branch name including the ticket ID. So, I'll rework this to note that the feature/name syntax should be used, and the ID should be included in the commit message(s). That should clean things up.